packages/accounts/src/kernel-zerodev/account.ts

```diff
@@ -0,0 +1,119 @@
+import type {Address} from "abitype";
+import {
+    concatHex,
+    encodeAbiParameters,
+    encodeFunctionData,
+    type FallbackTransport,
+    type Hex,
+    type Transport,
+} from "viem";
+import {type BaseSmartAccountParams, BaseSmartContractAccount,} from "@alchemy/aa-core/src/account/base";
+import {parseAbiParameters} from "abitype";
+import {SmartAccountSigner} from "@alchemy/aa-core/src/signer/types";
+import {BaseValidator, ValidatorMode} from "./validator/base";
+import {KernelAccountAbi} from "./abis/KernelAccountAbi";
+import {KernelFactoryAbi} from "./abis/KernelFactoryAbi";
+
+export interface KernelSmartAccountParams<
+    TTransport extends Transport | FallbackTransport = Transport
+> extends BaseSmartAccountParams<TTransport> {
+    owner: SmartAccountSigner;
+    factoryAddress: Address;
+    index?: bigint;
+    defaultValidator: BaseValidator
+    validator?: BaseValidator
+}
+
+export class KernelSmartContractAccount<
```

Comment:
do ZeroDev kernel account support batch executions?

Reply:
It does. But we will add implementation in later PR. as of now, it will show not supported